docs/api.md

```diff
@@ -26,6 +27,44 @@ Example: `hig.no/ifs/api/0`
 
 ## <a name="api"></a>APIs
 
+### <a name="add-comment"></a>Add comment
+
+Add a comment to a lecture.
+
+#### URL
+
+`/lectures/:lecture-id/comment`
```

Comment:
As I discussed in PR #18, I think it should be 'comments' in stead of 'comment' here.